rero_ils/modules/utils.py

```diff
@@ -161,3 +161,24 @@ def add_years(initial_date, years):
     except ValueError:
         return initial_date + (date(initial_date.year + years, 1, 1) -
                                date(initial_date.year, 1, 1))
+
+
+def get_schema_for_resource(resource):
```

Comment:
Good function, maybe we have to open a task to use this function in: IlsRecord, Holding, Loan, PatronTransactionEvent and PatronTRansaction.